src/util/handle-error.ts

```diff
@@ -23,22 +23,9 @@ export default function handleError(
       errorOutput('Authentication error. Run `now login` to log-in again.')
     );
   } else if ((<APIError>error).status === 429) {
```

Comment:
I realize you didn't change this code, but casting to `APIError` is a code smell.

Can you instead wrap this whole if statement with `if (typeof error.status === 'number')` which should narrow the type?

Reply:
That wouldn't narrow it down actually, user-defined type guards need to be used instead. I actually did spend close to an hour trying to remove the type casts from this file but ran into issues regarding the `NowError` and its subclasses (see `errors-ts.ts`). It was kind of a nightmare, and I ended up aborting mission.

IMO we need to re-think the generics of the `NowError` class and make it more… typeable, and DRY.

Reply:
Also there's a lot more casting in this file than just this one. Territory for a separate PR I think.